osu.Game/Screens/Play/Player.cs

```diff
@@ -54,8 +55,10 @@ public class Player : ScreenWithBeatmapBackground, IProvideCursor
 
         public int RestartCount;
 
+        private const double cursor_idle_delay = 1000;
+
         public CursorContainer Cursor => RulesetContainer.Cursor;
-        public bool ProvidingUserCursor => RulesetContainer?.Cursor != null && !RulesetContainer.HasReplayLoaded.Value;
+        public bool ProvidingUserCursor => (RulesetContainer?.Cursor != null || idleTracker?.MouseIdleTime > cursor_idle_delay) && !RulesetContainer.HasReplayLoaded.Value;
```

Comment:
shouldn't this be `&&` `<=`?

Reply:
It is [have a not null cursor provided] or [(do not have a not null cursor provided) and the time that mouse is idle is longer than `cursor_idle_delay`], and the outer menu cursor should be hidden if it is not playing a replay.